Setup/deploy-shield.sh

```diff
@@ -37,7 +37,7 @@ fi
 ########################################################################################################################
 
 UPSTREAM_DNS_SERVERS="$(grep -oP 'nameserver\s+\K.+' /etc/resolv.conf | cut -d, -f2- | paste -sd,)"
-if [ -z "$UPSTREAM_DNS_SERVERS" ]; then
+if [ "$UPSTREAM_DNS_SERVERS" = "127.0.0.53" ]; then
```

Comment:
what's this ip address?

Reply:
It is the IP address used by systemd-resolved when it is active (which is the case with Ubuntu 18.04).